ci/release-validate-deps.py

```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+
+if __name__ == '__main__':
+    dependencies_bzl = os.path.join('dependencies', 'graknlabs', 'dependencies.bzl')
+    snapshot_dependencies = []
+    with open(os.path.join(os.getenv("BUILD_WORKSPACE_DIRECTORY"), dependencies_bzl)) as dependencies_bzl_:
```

Comment:
@lolski better naming IMO would be `dependencies_bzl_fn` (filename) /`dependencies_bzl` (file object)

Reply:
Resolved in ad7e6c531b89c5711e31d4ab3e64b650463e56ad